tests/Traits/FieldsTrait.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_paragraphs\Traits;
+
+use Behat\Mink\Element\NodeElement;
+
+/**
+ * Helper methods to deal with fields.
+ */
+trait FieldsTrait {
+
+  /**
+   * Returns the n-th occurrence of a field in a page.
+   *
+   * @param string $field
+   *   The field element label.
+   * @param int $position
+   *   The position of the field amongst the ones with same label. Zero-based
+   *   index.
+   *
+   * @return \Behat\Mink\Element\NodeElement
+   *   The field element.
+   *
+   * @throws \Exception
+   *   Thrown when the specified occurrence of the field or the field itself is
+   *   not found.
+   */
+  protected function getNthField(string $field, int $position): NodeElement {
+    $fields = $this->getSession()->getPage()->findAll('named', ['field', $field]);
+
+    if (!$fields || !isset($fields[$position])) {
```

Comment:
We could probably improve the exception messages: One if we don't find any occurrence of the field and another if we don't find the nth occurrence. Not a hard requirement though.

Reply:
Well I didn't see much value on it, if the test failed it's because the element is not found at a specific position, which implies also the empty list, that's why I didn't put it.